Unix/regress

```diff
@@ -118,7 +118,29 @@ root=`dirname $0`
 ##==============================================================================
 cleanup() 
 { 
-    ./tools/auth_tests_cleanup.sh $1
+    ./tools/auth_tests_cleanup.sh $1 $2
+}
+
+##==============================================================================
+##
+## Create soft-link to $OMI_USER account
+##
+##==============================================================================
+createlink()
+{ 
+    username=$OMI_USER
+    new_output="/tmp/$USER-$current_output"
+
+    if [ "x$username" != "x" ]; then
+        userhome=`eval echo ~$username`
+        if [ -w $userhome ]; then
```

Comment:
Re-wrote function to:

createlink()
{ 
    username=$OMI_USER

# First, check if $OMI_USER's home directory is writable
    if [ "x$username" != "x" ]; then
        userhome=`eval echo ~$username`
        if [ -w $userhome ]; then
            new_output="$userhome/$USER-$current_output"
            rm -rf $new_output
            mkdir $new_output
            if [ $? -eq 0 ]; then
                ln -s $new_output $current_output
            fi
        fi
    fi

# If that's not available, try /tmp
    new_output_tmp="/tmp/$USER-$current_output"
    rm -rf $new_output
    mkdir $new_output
    if [ $? -eq 0 ]; then
        ln -s $new_output $current_output
    else
        echo "Unable to create softlink to $current_output.  Skipping..."
    fi
}